src/layout/components/NavLinks.tsx

```diff
@@ -0,0 +1,23 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMountainSun } from '@fortawesome/free-solid-svg-icons';
+
+type LinkProp = {
+   href: string;
+   text: string;
+};
+
+export const FooterLink: React.FC<LinkProp> = ({ href, text }) => (
+   <li className="footer-link-item text-cyan-500 hover:rounded-md duration-200 py-2 px-3 border-2 border-cyan-500">
+      <a href={href}>{text}</a>
+   </li>
+);
```

Comment:
Do this for the component bellow as well, since we don't need children:
```suggestion
export const FooterLink = ({ href, text }: LinkProp) => (
   <li className="footer-link-item text-cyan-500 hover:rounded-md duration-200 py-2 px-3 border-2 border-cyan-500">
      <a href={href}>{text}</a>
   </li>
);
```